backend/backend/api/routes.py

```diff
@@ -293,6 +295,11 @@ async def github_comment(repo, pr_number, request):
     return response
 
 
+async def github_pr(repo, pr_number):
+    response = await review_github_pr(Context(), repo, int(pr_number))
```

Comment:
The function `github_pr` converts `pr_number` to an integer using `int(pr_number)`. Ensure that `pr_number` is always a valid string representation of an integer, otherwise this will raise a ValueError.